mappings/net/minecraft/client/render/entity/model/EntityModels.mapping

```diff
@@ -0,0 +1,5 @@
+CLASS net/minecraft/class_5600 net/minecraft/client/render/entity/model/EntityModels
+	FIELD field_27543 FISH_PATTERN_DILATION Lnet/minecraft/class_5605;
+	FIELD field_27544 ARMOR_DILATION Lnet/minecraft/class_5605;
+	FIELD field_27545 HAT_DILATION Lnet/minecraft/class_5605;
+	METHOD method_32073 getModels ()Ljava/util/Map;
```

Comment:
```suggestion
	METHOD method_32073 getModelParts ()Ljava/util/Map;
```

Reply:
These are the data for the full entity models.